source/message24-decoded.txt

```diff
@@ -1,7 +1,7 @@
-x0
-ap x0 x0   =   x0
-ap x0 1   =   1
-ap x0 x0   =   x0
-ap x0 add   =   add
-ap x0 ap add 1   =   ap add 1
+id
+ap id x0   =   x0
```

Comment:
I think to be consistent the `x0`s here and two lines below would need to be changed to `id` too.

I'm not so sure we should change this one, though. IIRC most messages that define something aren't defining a generic variable like this.